### ISSUE TITLE
Latest beta missing all macOS bootable volumes

### ISSUE THREAD

**Issue by MarioG-X** [dakanji/RefindPlus#76]
**Describe the Bug:**
I thought you may want to know this if you are not aware of it. Beta built today. Menu shows all 4 Linux but missing all macOS volumes.

**To Reproduce:**
1. Build RP GOP as of 10/2/2021
2. Install into EFI
3. Boot

**Expected Behaviour:**
macOS volumes should display

**Environment:**
 - RefindPlus Version : Beta v0.13.2.AH Debug build as of Oct 2
   * Computer         : MacMini 2014
 - Affected OS        :  all Mac OS

**Log Files**
Loglevel 4:
[21t02t3849.log](https://github.com/dakanji/RefindPlus/files/7271922/21t02t3849.log)

loglevel 0:
[21t02t0632.log](https://github.com/dakanji/RefindPlus/files/7271906/21t02t0632.log)

**Comment by dakanji** [comment]
Thanks. I had noticed this but had a brain fog event and thought it was from a temp disabling of these in my config which it was not. Will push a fix later.

**Comment by MarioG-X** [comment]
Updated RP beta and tested Linux and macOS, AOK now. BTW, the reason I tested Oct 2 beta AH was because I experienced having to hit enter twice when selecting a boot volume. The first enter could be seen affecting the screen so it was recognized, and the second enter started the boot. This problem is fixed in the Oct 2 beta. Also looks like you are able to display the macOS system name, that's great. Thanks for the quick fix!
